pageserver/src/walredo/process.rs

```diff
@@ -136,7 +136,9 @@ impl WalRedoProcess {
                         Ok(0) => break Ok(()), // eof
                         Ok(num_bytes) => {
                             let output = String::from_utf8_lossy(&buf[..num_bytes]);
-                            error!(%output, "received output");
+							if !output.contains("LOG:") {
```

Comment:
you are using hard tabs here, while we indent with spaces. Not sure why rustfmt doesn't pick this up.